tests/integration/conftest.py

```diff
@@ -42,7 +42,7 @@ def app():
     app.config.update(
         {
             'DEBUG': True,
-            'WTF_CSRF_ENABLED': False
+            'WTF_CSRF_ENABLED': False,
```

Comment:
This is not needed. Just a leftover trying to make it work with `invenio-db[postgresql,versioning]==1.0.0b3`